glide.yaml

```diff
@@ -15,10 +15,12 @@ import:
   subpackages:
   - client
   - error
+- package: github.com/docker/distribution
```

Comment:
why this is this needed?

Reply:
something in kubernetes/utils (which is what i use for mounting and formating) required it.

Reply:
so its a dependency from kubernetes/utils? why does it need to be in our top level glide.yaml?

Reply:
i had to have it there to make glide happy. Let me remove it and try again to make sure